fsharp-backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -785,12 +844,18 @@ type BuiltInFn =
     previewable : Previewable
     deprecated : Deprecation
     sqlSpec : SqlSpec
+
+    // TODO: reevaluate usefulness of below comment. at least rephrase.
```

Comment:
I think this is still accurate, though the implementations are now in BackendOnlyStdlib, not LibBackend.